libs/dh/market-participant/actors/feature-organizations/src/lib/drawer/dh-organization-drawer.component.ts

```diff
@@ -99,23 +131,46 @@ export class DhOrganizationDrawerComponent {
   public open(organizationId: string): void {
     this.drawer?.open();
     this.loadOrganization(organizationId);
+    this.loadActors(organizationId);
   }
 
   onClose(): void {
     this.closed.emit();
   }
 
   private loadOrganization(id: string): void {
-    this.subscription?.unsubscribe();
+    this.organizationSubscription?.unsubscribe();
 
     this.getOrganizationByIdQuery$.setVariables({ id });
 
-    this.subscription = this.getOrganizationByIdQuery$.valueChanges
+    this.organizationSubscription = this.getOrganizationByIdQuery$.valueChanges
       .pipe(takeUntil(this.closed))
       .subscribe({
         next: (result) => {
           this.organization = { ...result.data.organizationById, organizationId: id };
         },
       });
   }
+
+  private loadActors(organizationId: string): void {
+    this.actorsSubscription?.unsubscribe();
+
+    this.getActorsByOrganizationIdQuery$.setVariables({ organizationId });
+
+    this.actorsSubscription = this.getActorsByOrganizationIdQuery$.valueChanges
+      .pipe(takeUntil(this.closed))
+      .subscribe({
+        next: (result) => {
+          this.actors = new WattTableDataSource<Actor>(
```

Comment:
suggestion: it's better to assign the new data to `this.actors.data` instead of creating a new `WattTableDataSource`. See "dh-actors-overview.component.ts#109".